conf/log4j.properties

```diff
@@ -12,8 +12,12 @@
 # May get overridden by System Property
 
 log4j.rootLogger=INFO, ${alluxio.logger.type}
+log4j.category.alluxio.master.audit=INFO, ${alluxio.master.audit.logger.type}
+log4j.additivity.alluxio.master.audit=false
 
+# NullAppender simply exists, and does not output message to any device.
 log4j.appender.Null=org.apache.log4j.varia.NullAppender
+log4j.appender.=org.apache.log4j.varia.NullAppender
```

Comment:
This can work, but I haven't been able to find documentation about using appender with empty string as name.